contracts/forwarder/Forwarder.sol

```diff
@@ -85,13 +88,32 @@ contract Forwarder is IForwarder {
         registerRequestTypeInternal(requestType);
     }
 
+    function registerDomainSeparator(string calldata name, string calldata version) external override {
+        uint256 chainId;
+        /* solhint-disable-next-line no-inline-assembly */
+        assembly { chainId := chainid() }
+
+        bytes memory domainValue = abi.encode(
+            keccak256(bytes(EIP712_DOMAIN_TYPE)),
+            keccak256(bytes(name)),
+            keccak256(bytes(version)),
+            chainId,
+            address(this));
+
+        bytes32 domainHash = keccak256(domainValue);
```

Comment:
There is GsnEip712Library::hashDomain, I suggest we reuse code to avoid duplication.

Reply:
the forwarder can't use "Gsn" methods - its a standalone contract.